lib/miq_automation_engine/engine/miq_ae_workspace.rb

```diff
@@ -99,6 +99,7 @@ def instantiate(uri, user, root = nil)
       $miq_ae_logger.info("Instantiating [#{uri}]") if root.nil?
       @ae_user = user
       @dom_search.ae_user = user
+      @dom_search.prepend_namespace  = root['prepend_namespace'] if root && root['prepend_namespace']
```

Comment:
@Fryguy it is stored in the root object. I am thinking if we should have a 
prepend_namespace(value) defined in MiqAeService that can be called as $evm.prepend_namespace("fred")

The only downside of this it can't be set externally by passing variables into Automate as part of the URL